webrender/Cargo.toml

```diff
@@ -17,6 +17,7 @@ replay = ["webrender_api/deserialize"]
 pathfinder = ["pathfinder_font_renderer", "pathfinder_gfx_utils", "pathfinder_partitioner", "pathfinder_path_utils"]
 serialize_program = ["webrender_build/serialize_program"]
 no_static_freetype = []
+headless_gfx = []
```

Comment:
do we really need a cargo feature for it? In the current WR it just drags the osmesa dependency and such, but with gfx-rs it's not needed. It could be purely a run-time switch.

Reply:
Thank you for the review! I will make the changes what you mentioned and I will update the PR.